pkg/action/install_test.go

```diff
@@ -798,3 +798,46 @@ func TestInstallWithSystemLabels(t *testing.T) {
 
 	is.Equal(fmt.Errorf("user supplied labels contains system reserved label name. System labels: %+v", driver.GetSystemLabels()), err)
 }
+
+func TestInstallWithTakeOwnershipOption(t *testing.T) {
+	is := assert.New(t)
+	req := require.New(t)
```

Comment:
The rest of the code seems to only use is and not require in this test file, maybe worth just removing req and using is.NoError on line 841